### PR TITLE
feat(ci): add mongo build

### PR DESCRIPTION
**Pull request by RealShadowNova**

### CODE REVIEW

**Review by WilsontheWolf — APPROVED** [review]
I don't really understand github actions that much but looks good enough. Just my one request is please test this before merging. Thanks

File: .github/workflows/continuous-delivery.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        uses: actions/checkout@v2
+      - name: Use Node.js 16
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+      - name: Restore CI Cache
+        uses: actions/cache@v2.1.6
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-16-${{ hashFiles('**/yarn.lock') }}
```
Comment (Chicken) [inline .github/workflows/continuous-delivery.yml:48]: Not needed. See [actions/setup-node](https://github.com/actions/setup-node#caching-packages-dependencies) docs.

File: .github/workflows/continuous-delivery.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+          echo "::add-matcher::.github/problemMatchers/eslint.json"
+    outputs:
+      commit_message: $( [ -z "${{ steps.pr_get_commit_message.outputs.pr_commit_message }}" ] && echo "${{ steps.push_get_commit_message.outputs.push_commit_message }}" || echo "${{ steps.pr_get_commit_message.outputs.pr_commit_message }}" )
+
+  Build_Mongo:
+    name: Publish Mongo Build
+    runs-on: ubuntu-latest
+    if: "!contains(needs.pre_ci.outputs.commit_message, '[skip ci]')"
+    needs: pre_ci
+    steps:
```
Comment (Chicken) [inline .github/workflows/continuous-delivery.yml:37]: Shouldn't this execute only when the path is `packages/mongo`. Maybe add something for that.

File: .github/workflows/continuous-delivery.yml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+          cd out
+          git add --all .
+          git config user.name "${GITHUB_ACTOR}"
+          git config user.email "${GITHUB_EMAIL}"
+          git commit -m "build(mongo): tsc build for ${GITHUB_SHA}" || true
+          git push origin $TARGET_BRANCH
+        env:
+          GITHUB_TOKEN: ${{ secrets.NOVA_TOKEN }}
+          GITHUB_ACTOR: RealShadowNova
+          GITHUB_EMAIL: real.shadow.nova@gmail.com
```
Comment (Chicken) [inline .github/workflows/continuous-delivery.yml:84]: Maybe use `github-actions <41898282+github-actions[bot]@users.noreply.github.com>` with `${{ secrets.GITHUB_TOKEN }}` instead.

File: .github/workflows/continuous-delivery.yml [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+          git clone $REPO out -b $TARGET_BRANCH
+
+          echo -e "\n# Remove any old files in the dist folder"
+          rm -rfv out/dist/*
+
+          echo -e "\n# Move the generated files to the newly-checked-out repo, to be committed and pushed"
+          rsync -vaI packages/mongo/package.json out/
+          rsync -vaI packages/mongo/LICENSE out/
+          rsync -vaI packages/mongo/README.md out/
+          rsync -vaI packages/mongo/dist/ out/dist
```
Comment (Chicken) [inline .github/workflows/continuous-delivery.yml:69]: Why rsync?

**Pull request merged**